js/foundation.dropdown.js

```diff
@@ -407,7 +417,14 @@ Dropdown.defaults = {
    * @type {boolean}
    * @default false
    */
-  closeOnClick: false
-}
+  closeOnClick: false,
+  /**
+   * Boolean to force overide the clicking of toggle link (href) to perform default action, on second touch event for mobile.
```

Comment:
I would rather say something like:
`If true, always follow the toggle link (or any default action) on click. Otherwise, wait for the second touch on touch devices`.